src/openrct2/peep/peep.c

```diff
@@ -1392,12 +1392,14 @@ static uint8 peep_get_action_sprite_type(rct_peep* peep)
 	}
 }
 
-void sub_693B58(rct_peep* peep){
+void peep_update_current_action_sprite_type(rct_peep* peep){
```

Comment:
I put them in - I agree that it's helpful to know whether a function could have original bugs.